demo/src/screens/componentScreens/TextFieldScreen.tsx

```diff
@@ -74,6 +80,7 @@ export default class TextFieldScreen extends Component {
           />
           <TextField
             placeholder="Placeholder"
+            placeholderTextColor={Colors.$textNeutralLight}
```

Comment:
Why are you setting a different placeholderColor here?

Reply:
I was just testing new token. Removed